File: class10/python/AP1003-MaiorMenor.py
# Maior e menor de um vetor
numeros = [4, 15, 8, 23, 16, 42, 7, 11, 9, 30]
maior = menor = numeros[0]
for x in numeros[1:]:
    if x > maior:
        maior = x
    if x < menor:
        menor = x
print("Vetor:", numeros)
print("Maior:", maior)
print("Menor:", menor)

File: class10/java/AP1003-MaiorMenor.java
public class MaiorMenor {
    public static void main(String[] args) {
        int[] numeros = {4, 15, 8, 23, 16, 42, 7, 11, 9, 30};
        int maior = numeros[0], menor = numeros[0];
        for (int i = 1; i < numeros.length; i++) {
            if (numeros[i] > maior) maior = numeros[i];
            if (numeros[i] < menor) menor = numeros[i];
        }
        System.out.println("Maior: " + maior);
        System.out.println("Menor: " + menor);
    }
}
